src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/custom.py

```diff
@@ -2322,13 +2334,17 @@ def create_deploy_webapp(cmd, name, location=None, sku=None, dryrun=False):  # p
             os.remove(zip_file_path)
         except OSError:
             pass
-    create_json.update({'app_url': url})
-    logger.warning("All done.")
-    return create_json
+    if logs:
+        logger.waring("Configure logging for a web app")
+        config_diagnostics(cmd, rg_name, name,
```

Comment:
like said, try create a common routine with a bit intelligence that to avoid overwrite users' existing logging config